src/echarts.ts

```diff
@@ -344,11 +345,23 @@ class ECharts extends Eventful {
             ) as any).__ECHARTS__DEFAULT__RENDERER__ || defaultRenderer;
         }
 
+        let defaultUseDirtyRect = false;
+        if (__DEV__) {
+            const devUseDirtyRect = ((
+                typeof window === 'undefined' ? global : window
+            ) as any).__ECHARTS__DEFAULT__USE_DIRTY_RECT__;
+
```

Comment:
We can reuse the logic of getting window or global in https://github.com/apache/incubator-echarts/blob/97b1473315f6f4dacc5b14fafce4b0a00f064722/src/echarts.ts#L344